package/Dockerfile.submariner-buildenv

```
@@ -0,0 +1,43 @@
+FROM fedora:33
+ARG TARGETPLATFORM
+
+# dnf cache and base packages, shared across all build sub-directories
+RUN dnf -y --setopt=install_weak_deps=0 --nodocs \
+           --installroot /output/base --releasever 33 \
+           install glibc bash glibc-minimal-langpack coreutils-single
+
+# gateway
+
+# iproute and iptables are used internally
+# libreswan provides IKE
+# procps-ng is needed for sysctl
+RUN cp -a /output/base /output/gateway && \
+    dnf -y --setopt=install_weak_deps=0 --nodocs \
+           --installroot /output/gateway --releasever 33 \
+           install libcurl-minimal \
+                   iproute iptables iptables-nft libreswan procps-ng && \
+    dnf -y --installroot /output/gateway --releasever 33 clean all
```

With the previous change this could be as simple as:
```suggestion
RUN INSTALL_ROOT=gateway /dnf_install.sh libcurl-minimal iproute iptables iptables-nft libreswan procps-ng
```